### ISSUE TITLE
Citation missing

### ISSUE THREAD

**Issue by sphinxnh** [dorianbrown/rank_bm25#13]
Hi,

I would like to cite your software artifact in my research paper as I used it to for ranking, but I did not find a CITATION file. Do you have a preferred way to cite your work?

Best,
nh

**Comment by dorianbrown** [comment]
Hey Nh,

Thanks for asking, to be honest I never realized people might want a way to cite it. I guess you're referencing [this](https://citation-file-format.github.io/) when you say a CITATION file? This shouldn't be too much work to get that set up, I'll look into it in the coming days.

Just out of curiosity, what kind of research paper are you writing? I originally created this repository for myself, so it's really neat to see other people getting some use out of it!

**Comment by sphinxnh** [comment]
Hi dorian,

thanks for you quick reply! I was not aware of the cff format as the scientific community has given emphasis on Latex and, therefore, bibtex style citation f.e. [https://github.com/explosion/spaCy/blob/master/CITATION](url)

I developed a design science artifact which searches abstract and citation databases (scopus, springernature, etc.), merges and classifies studied in the field of applied Information Systems in the lifecycle of a tunnel. I use your library to rank (tokenized) abstracts according to queries, which consist of descriptive keywords provided by domain experts. Then, I use the received scores as feature to train models (svc vs. logistic regression vs. boosting trees) in order to make multiple binary-classifications, such as: is the study relevant or not?

I started with TF and TF-IDF, but nothing beats your bm25 implementation. Keep up the good work 👍 

Best,
nh
